### PR TITLE
Simple authors change

### PR DESCRIPTION
**Pull request by andreazanetti**
### PR types

Others
### PR changes

Docs
### Describe

Update list of authors

### CODE REVIEW

**Review by jakpiase — CHANGES_REQUESTED** [review]
Please, add me (Jakub Piasecki) as well

File: AUTHORS.md [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+| arlesniak | Artur Lesniak |
+| arogowie-intel | Adam Osewski |
 | backyes | Yan-Fei Wang |
 | baiyfbupt | Yi-Fan Bai |
 | beckett1124 | Bin Qi |
 | ChengduoZH | Cheng-Duo Zhao|
 | chengxiaohua1105 | Xiao-Hua Cheng |
 | cxwangyi, yiwangbaidu, wangkuiyi | Yi Wang |
 | cxysteven | Xing-Yi Cheng |
+| ddokupil | Darius Dokupil |
```
Comment (arlesniak) [inline AUTHORS.md]: Please correct the typo: Dariusz,

File: AUTHORS.md [hunk 2]
```diff
@@ -42,14 +46,13 @@
...
 | pengli09 | Peng Li |
+| pmajchrzak |Piotr Majchrzak |
 | pkuyym | Ya-Ming Yang |
 | pzelazko-intel | Pawel Zelazko |
 | [pawelpiotrowicz](https://raw.githubusercontent.com/jczaja/Paddle/paddle-poland-team/doc/images/paddle_poland_team.jpg)  | Pawel Piotrowicz |
 | QiJune | Jun Qi |
 | qingqing01 | Qing-Qing Dang |
 | reyoung | Yang Yu |
-| [Sand3r-](https://raw.githubusercontent.com/jczaja/Paddle/paddle-poland-team/doc/images/paddle_poland_team.jpg)| Michal Gallus |
-| [sfraczek](https://raw.githubusercontent.com/jczaja/Paddle/paddle-poland-team/doc/images/paddle_poland_team.jpg)| Sylwester Fraczek |
```
Comment (lidanqing-vv) [inline AUTHORS.md]: These names are removed because they left Intel or they left PaddlePaddle? But they also contributed some code in PaddlePaddle

Comment (lidanqing-vv) [reply AUTHORS.md]: I confirmed with Luotao. @andreazanetti  Do not delete their names since they contributed before.

**Review by jakpiase — APPROVED** [review]
LGTM!

**Review by arlesniak — APPROVED** [review]
LGTM

**Review by luotao1 — APPROVED** [review]

**Pull request merged**